watertap/unit_models/pressure_exchanger.py

```diff
@@ -210,134 +241,203 @@ def build(self):
+            doc="Work transferred to brine side fluid (should be negative)",
         )
         def work(b, t):
             return b.properties_in[t].flow_vol * b.deltaP[t]
 
-        # Build control volume for low pressure side
-        self.low_pressure_side = ControlVolume0DBlock(
+        # Build control volume for feed side
+        self.feed_side = ControlVolume0DBlock(
             dynamic=False,
             has_holdup=False,
             property_package=self.config.property_package,
             property_package_args=self.config.property_package_args,
         )
 
-        self.low_pressure_side.add_state_blocks(has_phase_equilibrium=False)
+        self.feed_side.add_state_blocks(has_phase_equilibrium=False)
 
-        self.low_pressure_side.add_material_balances(
+        self.feed_side.add_material_balances(
             balance_type=self.config.material_balance_type,
-            has_mass_transfer=self.config.has_mass_transfer,
+            has_mass_transfer=self.config.has_mixing,
         )
 
-        self.low_pressure_side.add_energy_balances(
+        self.feed_side.add_energy_balances(
             balance_type=self.config.energy_balance_type,
             has_enthalpy_transfer=False,
         )
 
         if self.config.is_isothermal:
-            self.low_pressure_side.add_isothermal_assumption()
+            self.feed_side.add_isothermal_assumption()
 
-        self.low_pressure_side.add_momentum_balances(
+        self.feed_side.add_momentum_balances(
             balance_type=self.config.momentum_balance_type, has_pressure_change=True
         )
 
-        @self.low_pressure_side.Expression(
+        @self.feed_side.Expression(
             self.flowsheet().config.time,
-            doc="Work transferred to low pressure side fluid",
+            doc="Work transferred to feed side fluid",
         )
         def work(b, t):  # pylint: disable=function-redefined
             return b.properties_in[t].flow_vol * b.deltaP[t]
 
         # Add Ports
-        self.add_inlet_port(name="high_pressure_inlet", block=self.high_pressure_side)
-        self.add_outlet_port(name="high_pressure_outlet", block=self.high_pressure_side)
-        self.add_inlet_port(name="low_pressure_inlet", block=self.low_pressure_side)
-        self.add_outlet_port(name="low_pressure_outlet", block=self.low_pressure_side)
+        self.add_inlet_port(name="brine_inlet", block=self.brine_side)
+        self.add_outlet_port(name="brine_outlet", block=self.brine_side)
+        self.add_inlet_port(name="feed_inlet", block=self.feed_side)
+        self.add_outlet_port(name="feed_outlet", block=self.feed_side)
 
         # Performance equations
         @self.Constraint(self.flowsheet().config.time, doc="Pressure transfer")
         def eq_pressure_transfer(b, t):
             return (
-                b.low_pressure_side.deltaP[t]
-                == b.efficiency_pressure_exchanger[t] * -b.high_pressure_side.deltaP[t]
+                b.feed_side.deltaP[t]
```

Comment:
I suggest implementing a more precise efficiency calculation method for PressureExchangeType.high_pressure_difference option, defining efficiency as the ratio of energy output to energy input. This approach effectively incorporates the effects of HPD and leakage into efficiency

Reply:
@ElmiraShamlou the current definition of  `efficiency` in `eq_pressure_transfer` is exactly same as ratio of energy output to energy input. I've tried to directly define the `efficiency` based on the energy output/energy input and get exact same results. Though, it is not so evident, ΔP_FS=-ηΔP_BS is same as energy output/ energy input

Reply:
Thanks @luohezhiming . The current definition of efficiency is equivalent to energy ratios only when there's no leakage and the flow rates on both the feed and brine sides are the same. You got the same results from both methods because the leakage was either very small or zero. But if leakage increases, it will have a bigger effect on efficiency. We can include the leakage inefficiency in a complete energy equation that incorporates both pressure and flow rates.

Reply:
I redefine the efficiency as the ratio of energy_in over energy_out, but I find the solution results will change for the `no_mass_transfer` case. Since a lot of flowsheet use this basic pressure exchanger without mass transfer, @TimBartholome, I want to confirm with you should we change this efficiency equation? If we do, then all flowsheets using the pressure exchanger will have different results.